docs/docs/testing-css-in-js.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Testing CSS-in-JS"
+---
+
+Popular CSS-in-JS libraries like [styled-components](https://github.com/styled-components/styled-components) or [emotion](https://github.com/emotion-js/emotion) can also be tested with the help of [jest-styled-components](https://github.com/styled-components/jest-styled-components) or [jest-emotion](https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion) respectively. These packages improve the snapshot testing experience and are a great way to help avoid unintended changes to your website's UI. Please refer to your package's documentation to see if it also offers testing capabilities.
+
+By default normal snapshots show the generated class names (which we didn't set) and no styling information. When changing the styles you'll only see the diff of some cryptic class names. That's why we should use the above mentioned *snapshotSerializers*.
+
+For our example we'll use emotion. The testing utitlities of emotion and glamor are largely based on [jest-styled-components](https://github.com/styled-components/jest-styled-components) so they have a similar usage. Please have a look at the testing section of your library to follow along.
```

Comment:
Sorry for doing some of these as comments instead of editing the PR... I guess I started this way so I'll finish!

```diff
- For our example
+ For this example
```


```diff
- utitlities
+ utilities
```